frontend/js/src/explore/fresh-releases/components/ReleaseTimeline.tsx

```diff
@@ -100,26 +173,16 @@ function createMarks(
       .map((_, index, arr) =>
         arr.slice(0, index + 1).reduce((prev, curr) => prev + curr)
       );
-  }
 
-  if (sortDirection === "descend") {
-    dataArr.reverse();
-    percentArr = percentArr.reverse().map((v) => (v <= 100 ? 100 - v : 0));
-  }
+    if (sortDirection === "descend") {
+      dataArr.reverse();
+      percentArr = percentArr.reverse().map((v) => (v <= 100 ? 100 - v : 0));
+    }
 
-  /**
-   * We want the timeline dates or marks to start where the grid starts.
-   * So the 0% should always have the first date. Therefore we use unshift(0) here.
-   * With the same logic, we don't want the last date to be at 100% because
-   * that will mean we're at the bottom of the grid.
-   * The last date should start before 100%. That explains the pop().
-   * For descending sort, the reverse computation above possibly already ensures that
-   * the percentArr starts with 0 and ends with a non-100% value, which is desired.
-   * Hence, we add a check to skip the unshift(0) and pop() operations in that case.
```

Comment:
I think this comment was useful to understand  the last block of code with the unshift and pop.
Did you remove it by accident?

I'm not opposed to rephrasing the comment if you want, but it was there for a reason.